app/src/main/java/io/appium/uiautomator2/server/ServerInstrumentation.java

```diff
@@ -3,15 +3,20 @@
 import android.content.Context;
 import android.os.Looper;
 import android.os.PowerManager;
+import android.os.RemoteException;
 
+import io.appium.uiautomator2.common.exceptions.SessionRemovedException;
 import io.appium.uiautomator2.utils.Logger;
 
+import static io.appium.uiautomator2.utils.Device.getUiDevice;
+
 public class ServerInstrumentation {
     private static ServerInstrumentation instance;
     private static Context context;
     private HttpdThread serverThread;
     private PowerManager.WakeLock wakeLock;
     private int serverPort = 8080;
+    private static boolean isStopServer;
```

Comment:
What does this track? It seems to only be set once, to `True`, and then can never be anything else.